### ISSUE TITLE
Installation on Ubuntu-linux is flaky

### ISSUE THREAD

**Issue by dccote** [DCC-Lab/PyTissueOptics#116]
**Describe the bug**
It is complex to install al the required libraries on a fresh ubuntu 22.04 distribution.

It may be possible to install via `apt install` the required libraries. So far, no success: unable to fully install mayavi.

**To Reproduce**
pip install -r requirements is not sufficient: it fails 

**Expected behavior**
Should install.

**Additional context**

**Comment by Mateuszq28** [comment]
Can somebody provide the exact requirements.txt package list for conda/pip that will work? I still get an error when after typing:

``
python setup.py build
python setup.py install
python -m pytissueoptics --help (same for python pytissueoptics)
``

  File "/home/miki/anaconda3/envs/sim3/lib/python3.8/site-packages/pyface-8.0.0-py3.8.egg/pyface/base_toolkit.py", line 165, in __init__
    raise NotImplementedError(msg % (toolkit, package, name))
NotImplementedError: the null tvtk.pyface.ui.null backend doesn't implement scene_editor:SceneEditor

I used Ubuntu WSL2. On a Windows same error arrises.

**Comment by Mateuszq28** [comment]
Maybe there is a problem with mayavi when it uses deprecated eggs to install. Or these eggs uses links to install dependencies and those dependencies have been updated to version that are not compatibile with mayavi itself. I tried to install mayavi directly from github site and still an error arrises. No matter which version of mayavi I use, it still crash in that point. I tried 4.8.1, 4.8.2, 4.8.3.dev.
